ci/jvm/build-test-jvm.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+# Clone virgil
+cd ..
+git clone https://github.com/titzer/virgil
+cd virgil
+
+# Set up latest version of virgil
+export PATH=$PATH:"$PWD/bin:$PWD/bin/dev:$PWD/test/config"
+./test/configure
+make
+
+# Set up wizard
+cd ../wizard-engine
+make -j
+
+# Install OCaml for specification tests
+sudo apt install -y ocaml ocamlbuild
```

Comment:
The `wasm-spec` repo seems to be pretty sensitive to the ocaml version. Do we need to specify that too?

Reply:
The `Makefile` in the `wasm-spec` repo says anything > 4.08 is fine. Ubuntu 22.04 has OCaml 4.13.1, so it should be okay unless they bump the min version requirement for OCaml in `wasm-spec`.

Reply:
In theory, yes. In practice I have hit several incompatibilities (but mostly with proposal repos, which are forks of the main spec). I guess it's OK for now.

Reply:
I haven't added proposal tests as I wasn't sure how to do that. I tried getting `opam` or whatever to work to get the latest OCaml version, but it was gloriously failing on the CI (while working on my local machine), so I gave up and just got the package manager versions of OCaml and ocamlbuild.